### ISSUE TITLE
python bindings: drop requirements on python2 on Fedora

### ISSUE THREAD

**Issue by llohse** [tschoonj/xraylib#113]
The xraylib-python package currently has an explicit dependency on python2 and the python2 version of numpy.

Fedora is deprecating Python 2 modules as of Fedora 30 [1]. The current plan is to retire Python 2 with Fedora 32 [2].

To solve this I suggest either
 - drop the explicit dependency on python2 and numpy for the xraylib-python package
 - or create a separate xraylib-python3 package for fedora that does not depend on python2 packages.

[1] https://fedoraproject.org/wiki/Changes/Mass_Python_2_Package_Removal
[2] https://fedoraproject.org/wiki/FinalizingFedoraSwitchtoPython3

**Comment by tschoonj** [comment]
Thanks for bringing this to my attention.

I will update my SPEC file accordingly...

**Comment by tschoonj** [comment]
See #121.

I have extracted the python2 bindings into their own subpackage. Starting with Fedora 31, they will no longer be built.

This may still need changes when CentOS 8 is released.

**Comment by llohse** [comment]
Somehow this did not work. Also there seems to be only a single xraylib-python package.

```
# dnf repoquery --depends --resolve xraylib-python
Last metadata expiration check: 0:57:13 ago on Tue 03 Dec 2019 09:25:21 AM CET.
glibc-0:2.30-7.fc31.i686
glibc-0:2.30-7.fc31.x86_64
libgcc-0:9.2.1-1.fc31.x86_64
libgomp-0:9.2.1-1.fc31.x86_64
python2-0:2.7.17-1.fc31.i686
python2-0:2.7.17-1.fc31.x86_64
python2-numpy-1:1.16.4-5.fc31.i686
python2-numpy-1:1.16.4-5.fc31.x86_64
python3-0:3.7.5-1.fc31.i686
python3-0:3.7.5-1.fc31.x86_64
python3-numpy-1:1.17.4-2.fc31.x86_64
xraylib-0:3.3.0-1.fc31.x86_64
# cat /etc/fedora-release 
Fedora release 31 (Thirty One)
```

Similarly on Centos8 
```
# dnf repoquery --depends xraylib-python
Last metadata expiration check: 1:28:23 ago on Tue 03 Dec 2019 08:58:19 AM CET.
/usr/libexec/platform-python
libc.so.6(GLIBC_2.4)(64bit)
libgcc_s.so.1()(64bit)
libgcc_s.so.1(GCC_3.3.1)(64bit)
libgomp.so.1()(64bit)
libgomp.so.1(GOMP_1.0)(64bit)
libgomp.so.1(GOMP_4.0)(64bit)
libgomp.so.1(OMP_1.0)(64bit)
libm.so.6()(64bit)
libpthread.so.0()(64bit)
libxrl.so.7()(64bit)
python(abi) = 2.7
python(abi) = 3.6
python2
python2-numpy
python3
python3-numpy
rtld(GNU_HASH)
xraylib
# cat /etc/centos-release
CentOS Linux release 8.0.1905 (Core) 
```

**Comment by tschoonj** [comment]
Apologies, this fix will only go into the 4.0.0 rpm packages, which I hope to release before Christmas...

**Comment by llohse** [comment]
Fair enough. Thanks a lot!
